generators/kubernetes/templates/registry/_jhipster-registry.yml

```diff
@@ -84,17 +84,13 @@ spec:
         # Registry configuration
         - name: SPRING_PROFILES_ACTIVE
           value: prod,native
-        - name: SECURITY_USER_PASSWORD
+        - name: JHIPSTER_REGISTRY_PASSWORD
           valueFrom:
             secretKeyRef:
               name: registry-secret
               key: registry-admin-password
         - name: JHIPSTER_SECURITY_AUTHENTICATION_JWT_SECRET
           value: my-secret-token-to-change-in-production
-        - name: EUREKA_CLIENT_FETCH_REGISTRY
-          value: 'true'
-        - name: EUREKA_CLIENT_REGISTER_WITH_EUREKA
-          value: 'true'
```

Comment:
I referred [this](http://blog.abhijitsarkar.org/technical/netflix-eureka/). I have changed couple of things in this one, will give a pull request with my changes. I can confirm that peer-aware mode is working fine with the changes. I have done multiple rounds of testing in OpenShift with plenty of combinations that I could think of. Will test K8s tonight. If possible, you may pls try out various combinations with the changes as well.

But by the way, sync between the instances will take few seconds as it is determined by the parameter EUREKA_SERVER_PEER_EUREKA_NODES_UPDATE_INTERVAL_MS. Generally to avoid too many updates, it will be kept somewhere between 15 and 45s.